code/modules/cm_marines/overwatch.dm

```diff
@@ -900,20 +900,12 @@
 	faction = FACTION_CLF
 /obj/structure/machinery/computer/overwatch/upp
 	faction = FACTION_UPP
+	minimap_type = MINIMAP_FLAG_UPP
 	command_channel_key = "#v"
 	ui_theme = "crtupp"
 	possible_options = list("UPP" = "crtupp", "Green" = "crtgreen", "Yellow" = "crtyellow", "Red" = "crtred")
 	chosen_theme = list("UPP", "Green", "Yellow", "Red")
 
-/obj/structure/machinery/computer/overwatch/upp/Initialize()
-	. = ..()
-
-	current_orbital_cannon = null
```

Comment:
we probably want to keep this to avoid the upp getting control over the almayer's ob, no?